core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1468,6 +1468,13 @@ private static RelDataType deriveTypeMapConcat(SqlOperatorBinding opBinding) {
           ReturnTypes.TO_MAP_VALUES_NULLABLE,
           OperandTypes.MAP);
 
+  /** The "MAP_CONTAINS_KEY(map, key)" function. */
+  @LibraryOperator(libraries = {SPARK})
+  public static final SqlFunction MAP_CONTAINS_KEYS =
```

Comment:
`MAP_CONTAINS_KEYS` -> `MAP_CONTAINS_KEY`?

Reply:
done, thank you